### ISSUE TITLE
Video issues after latest update on Arch GNU/Linux

### ISSUE THREAD

**Issue by r3k2** [swaywm/sway#7999]
### Please fill out the following:
- latest version from Arch that came out today
- Video was broken and skipping frames on 3 diff computers, so downgraded to 1.8.1 working as before now.

No log, system crashed and had to force reboot on 3 occasions, 2 on my main PC and 1 on one of my laptops.
I do not have another OS so have to downgrade like it or not I only run GNU/Linux everywhere so no way for me to 
do anything without downgrading first and then submitting the issue.

- **Description:**
  - did my regular daily pacman -Syu 
  - I restarted, it seemed ok, until I started to play a movie that I was watching just fine the night before with mpv
  - eventually it crashed, had to hard reboot... I scp the movie to my laptop, same issue but did not crash.
  - I downgraded on all my computers to 1.8.1 now works great.
  
Sorry I can't be of more help with logs and such, I need my computers for work and had to downgrade ASAP.
Mainly wondering if more people have similar issues with video playing and eventually crashing, either or both.

**Comment by flyingfishflash** [comment]
I'm not clear on the root cause, but on my Lemur Pro intel i915 11Gen, for some reason the video display hangs on a gray screen with a non-responsive mouse. If I do nothing it will never progress. Yet, if I start to continuously move the mouse the display loads in about 20 sec... no idea. Trying to figure out how to capture some debugging.

Also when it does load some of the start up items haven't executed. Also on Arch.

**Comment by atmorling** [comment]
Experienced the same issue as @flyingfishflash  on a Thinkbook Yoga i7. Also had noticeably worse performance tiling (for example, opening a new app that caused an existing app to resize).
Unfortunately couldn't see anything that looked like a problem in sway's debug output, but will try to gather some more info ASAP.

Downgrading to 1.8.1 fixed it immediately.

**Comment by atmorling** [comment]
[Logs and cofig here](https://gist.github.com/atmorling/932d8364c6c19a57c8b7b55ad34cd646) 

Workflow was: 
- run sway from tty
- wait on grey screen a while
- wiggle mouse until the expected desktop appeared 
- open some apps (alacritty, firefox) and exit.

I've included a log of the same flow (sans grey screen and mouse wiggles) in 1.8 in case it's useful.

Comparing the two, things go wrong around [here](https://gist.github.com/atmorling/932d8364c6c19a57c8b7b55ad34cd646#file-swaydebug_1-9-log-L3036) with the below line printing while everything appears to hang. 

```
[wlr] [backend/drm/atomic.c:239] Failed to acquire FB for plane 94
```

I wasn't able to reproduce the performance issues, everything else seemed fine once 1.9 actually loaded.

**Comment by wooque** [comment]
Probably some startup application, I had similar issue, grey screen temp hanging on startup, in my case it was because of kanshi, some had similar problems because of alacrity.

https://www.reddit.com/r/debian/comments/199o6s3/debian_testingtrixie_and_sway_caution_today/

**Comment by emersion** [comment]
This seems to be a duplicate of https://gitlab.freedesktop.org/wlroots/wlroots/-/issues/3734

**Comment by emersion** [comment]
Tentative fix: https://gitlab.freedesktop.org/wlroots/wlroots/-/merge_requests/4577

**Comment by emersion** [comment]
The fix will be included in wlroots 0.17.2.
